deluge/plugins/AutoAdd/deluge/plugins/autoadd/core.py

```diff
@@ -253,7 +253,7 @@ def update_watchdir(self, watchdir_id):
                 continue
             else:
                 ext = os.path.splitext(filename)[1]
-                if ext == ".torrent":
+                if ext == ".torrent" or ".TORRENT":
```

Comment:
Yeah, that is a good solution. Your code could also be written as
`if ext  in [".torrent", ".TORRENT"]`